Countly.d.ts

```diff
@@ -1213,3 +1218,41 @@ declare module "countly-sdk-react-native-bridge/CountlyConfig" {
 
   export default CountlyConfig;
 }
+
+declare module "countly-sdk-react-native-bridge/lib/configuration_interfaces/countly_config_apm" {
```

Comment:
idt you need to declare it. Cant u add a nested interface on countly config?

Reply:
done.